lib/matplotlib/tests/test_lines.py

```diff
@@ -7,7 +7,7 @@
 import six
 
 import nose
-from nose.tools import assert_true, assert_raises
+from nose.tools import assert_true, assert_raises, assert_raises_regexp
```

Comment:
I think dropping 2.6 would put us at the leading edge of the de-adoption curve, we sure we want to be there?

Also raises the question of when we can drop 2.7 as well :smiling_imp: